cirq-core/cirq/experiments/__init__.py

```diff
@@ -82,3 +82,6 @@
     parallel_two_qubit_xeb as parallel_two_qubit_xeb,
     run_rb_and_xeb as run_rb_and_xeb,
 )
+
+
+from cirq.experiments.z_phase_calibration import z_phase_calibration_workflow, calibrate_z_phases
```

Comment:
It's a little redundant to have both `calibrate_z_phases` and `z_phase_calibration_workflow`. Can we just have one thing with the functionality of `z_phase_calibration_workflow`?

Reply:
are you sure?